web/public/styles.css

```
@@ -85,4 +87,8 @@ h6 {
     .examples-grid {
         grid-template-columns: repeat(2, minmax(0, 1fr));
     }
+    .example-card--full {
+        /* GRID take up 2 columns*/
```

Stylelint (17.2.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)